modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path

```diff
@@ -365,6 +365,123 @@
 	<td></td>
 </tr>
 
+<tr>
+	<td>RULE_BUILDER_FORM_VIEW_NAME</td>
+	<td>//input[contains(@placeholder,'Untitled Form View')]</td>
+	<td></td>
+</tr>
+
+<!--TABS-->
+<tr>
+	<td>RULES_TAB</td>
+	<td>//button[@id='rules']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FIELDS_TAB</td>
+	<td>//button[@id='fields']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_NAME_TEXT_FIELD</td>
+	<td>//input[@placeholder='Untitled Rule']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_ADD_BUTTON</td>
+	<td>//div[contains(@class,'multi-panel-sidebar')]//button[@class='btn btn-monospaced btn-primary']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_BUILDER_NAME</td>
+	<td>//input[@placeholder='Untitled Rule']</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>CREATE_NEW_RULE_BUTTON</td>
+	<td>//button[contains(text(),'Create New Rule')]</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>EDIT_RULE_COLLAPSE</td>
+	<td>//button[contains(@class,'collapse') and contains(@aria-expanded,'false')]//span[contains(.,'${ruleName}')]</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>EDIT_RULE_EXPANDED</td>
+	<td>//button[contains(@class,'collapse') and contains(@aria-expanded,'true')]//span[contains(.,'${ruleName}')]</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>EDIT_RULE</td>
```

Comment:
Hi @calvinctvo , please update this selector to:

`//div[contains(@class,'show')]//div[contains(@class,'panel-body')]//button[contains(@class,'btn-unstyled')]`

So you don't need to use a parameter, since you are pointing to a visible child element.